README.rst

```diff
@@ -248,11 +222,6 @@ TODO
     :scale: 100%
     :target: https://travis-ci.com/pyccel/pyccel
```

Comment:
This line also needs to be updated.

Reply:
I didn't succeed in changing the badge to point to the GitHub actions link, I found this link https://docs.github.com/en/free-pro-team@latest/actions/managing-workflow-runs/adding-a-workflow-status-badge, but it doesn't seem to work for me

Reply:
Could you post every link that you have tried so far?

Reply:
By the way, have you already tried this?
https://github.com/pyccel/pyccel/workflows/.github/workflows/Github_pytest.yml/badge.svg

Reply:
Probably better:
https://github.com/pyccel/pyccel/workflows/.github/workflows/Github_pytest.yml/badge.svg?branch=master

Reply:
If we create a new workflow which only runs on pushes to the master branch and keep the current workflow for pull requests to the master branch then we can link to the badge for the new workflow